package.json

```diff
@@ -8,6 +8,8 @@
     "@ant-design/icons": "^4.2.2",
     "@auth0/auth0-react": "^1.6.0",
     "@craco/craco": "^5.9.0",
+    "@okta/okta-react": "^3.0.2",
+    "@okta/okta-signin-widget": "^4.1.2",
```

Comment:
Glad you caught that!  That will save on future errors until the merge is  complete.